clients/include/hipblaslt_arguments.hpp

```diff
@@ -143,6 +143,7 @@ struct Arguments
     bool                     c_equal_d;
     bool                     HMM;
     bool                     use_e;
+    hipDataType              aux_type;
```

Comment:
better to move to after "hipDataType          scale_type;"

Reply:
but `aux_type` is part of the epilogue?

Reply:
Or move to after "bias_type"